wpe/src/main/cpp/browser/entrypoints.cpp

```diff
@@ -253,7 +260,8 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM* vm, void* reserved)
             { "touchEvent",               "(IJIFF)V",                                   reinterpret_cast<void*>(touchEvent) },
             { "setZoomLevel",             "(ID)V",                                      reinterpret_cast<void*>(setZoomLevel) },
             { "setInputMethodContent",    "(IC)V",                                      reinterpret_cast<void*>(setInputMethodContent) },
-            { "deleteInputMethodContent", "(II)V",                                      reinterpret_cast<void*>(deleteInputMethodContent) }
+            { "deleteInputMethodContent", "(II)V",                                      reinterpret_cast<void*>(deleteInputMethodContent) },
+            { "requestExitFullscreenMode", "(I)V",                                      reinterpret_cast<void*>(requestExitFullscreenMode) }
```

Comment:
Use Android Studio reformat tool (Ctrl+Alt+Enter), it will automatically realign all columns.

Reply:
fixed